client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/boost-site-performance.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@automattic/components';
+import { Icon, help } from '@wordpress/icons';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { jetpackBoostDesktopIcon, jetpackBoostMobileIcon } from '../../icons';
+import { getBoostRating, getBoostRatingClass } from '../utils';
+import ExpandedCard from './expanded-card';
+import type { BoostData } from '../types';
+
+interface Props {
+	boostData: BoostData;
+	hasBoost: boolean;
+}
+
+export default function BoostSitePerformance( { boostData, hasBoost }: Props ) {
+	const translate = useTranslate();
+
+	const { overall: overallScore, mobile: mobileScore, desktop: desktopScore } = boostData;
+
+	const components = {
+		strong: <strong></strong>,
+	};
+
+	return (
+		<ExpandedCard
+			header={ translate( 'Boost site performance' ) }
+			isEnabled={ !! hasBoost }
+			emptyContent={ translate(
+				'Add {{strong}}Boost{{/strong}} to see your site performance scores',
+				{
+					components,
+				}
+			) }
+		>
```

Comment:
The figma drawing for this shows that the box should be transparent and the text is center, instead of left justified.

![image](https://user-images.githubusercontent.com/2293077/224132382-79693dd9-468e-45d8-9a41-e7f6700f0000.png)

Reply:
I fixed the background color - https://github.com/Automattic/wp-calypso/pull/74038/commits/ba7f3668ddece8f21e3368b1b64b77cd5e3af6ea. It is already center-aligned.

Reply:
The text still looks left justified to me:
![image](https://user-images.githubusercontent.com/2293077/225053119-66978b7f-3a49-4929-8dc0-74c64e10af1f.png)

A `text-align: center` should do the trick.

Reply:
Thanks. Fixed now - https://github.com/Automattic/wp-calypso/pull/74038/commits/5c805c102e257106773d353578475cdce89a0dd0

<img width="337" alt="Screenshot 2023-03-15 at 4 10 36 PM" src="https://user-images.githubusercontent.com/10586875/225285296-d2ac0ce2-9db6-4383-a34a-61ea92b6aba3.png">